README.md

```diff
@@ -1,4 +1,4 @@
-# COSC499-Team-Git
+# Feature/ Word-Counting
```

Comment:
This is not needed here. The README.md file is not used to describe a specific to a branch. It is used to describe the repository, so we want to keep it general

Reply:
Sorry for the miss-understood for the ReadMe file.